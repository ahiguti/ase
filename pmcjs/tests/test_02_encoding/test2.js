
Perl.Use("TextTest");
var tt = Perl.Functions["TextTest"];
tt.setbinmode();
var txt = "あいうえお";
var s = tt.print(txt);
dump(s);
print();
var txtwrap = Perl.NewScalar();
txtwrap.SetText(txt);
s = tt.print(txtwrap);
dump(s);
print();
var binwrap = Perl.NewScalar();
binwrap.SetBinary(txt);
s = tt.print(binwrap);
dump(s);
print();
