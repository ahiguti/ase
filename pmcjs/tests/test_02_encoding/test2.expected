isutf: あいうえお
isutf: あいうえお
not: BDFHJ
3042 3044 3046 3048 304a
3042 3044 3046 3048 304a
42 44 46 48 4a
